src/main/java/com/opsmatters/newrelic/api/model/graphql/EntityLookupResponse.java

```diff
@@ -232,4 +250,75 @@ public void setNextCursor(String nextCursor) {
             this.nextCursor = nextCursor;
         }
     }
+
+    public class Settings {
+        private Double apdexTarget;
+
+        public Double getApdexTarget() {
+            return apdexTarget;
+        }
+
+        public void setApdexTarget(Double apdexTarget) {
+            this.apdexTarget = apdexTarget;
+        }
+
+    }
+
+    public class ApmSummary {
+        private Double apdexScore;
+        private Double errorRate;
+        private Integer hostCount;
+        private Integer instanceCount;
+        private Double responseTimeAverage;
+        private Double throughput;
+
+        public Double getApdexScore() {
+            return apdexScore;
+        }
+
+        public void setApdexScore(Double apdexScore) {
+            this.apdexScore = apdexScore;
+        }
+
+        public Double getErrorRate() {
+            return errorRate;
+        }
+
+        public void setErrorRate(Double errorRate) {
+            this.errorRate = errorRate;
+        }
+
+        public Integer getHostCount() {
+            return hostCount;
+        }
+
+        public void setHostCount(Integer hostCount) {
+            this.hostCount = hostCount;
+        }
+
+        public Integer getInstanceCount() {
+            return instanceCount;
+        }
+
+        public void setInstanceCount(Integer instanceCount) {
+            this.instanceCount = instanceCount;
+        }
+
+        public Double getResponseTimeAverage() {
+            return responseTimeAverage;
+        }
+
+        public void setResponseTimeAverage(Double responseTimeAverage) {
```

Comment:
Is the response time average in the apm summary?

Reply:
No, but ApmApplicationEntityOutline is missing response time, so I want to compare it.